src/router/index.ts

```diff
@@ -1,17 +1,18 @@
 import { createRouter, createWebHistory } from 'vue-router'
 import TitleView from '../pages/title-view.vue'
+import { RouterName } from '../types/enums/RouterName'
 
 const router = createRouter({
   history: createWebHistory(import.meta.env.BASE_URL),
   routes: [
     {
-      path: '/',
-      name: 'home',
+      path: `/`,
+      name: RouterName.HOME,
       component: TitleView
     },
     {
-      path: '/task',
-      name: 'task',
+      path: `/${RouterName.TASK}`,
```

Comment:
with what you declared in src/types/enums/RouterName.ts
would the path be `//task`?

Reply:
noted